apigentools/cli.py

```diff
@@ -20,6 +20,10 @@ def get_cli_parser():
     p = argparse.ArgumentParser(
         description="Manipulate OpenAPI specs and generate code using openapi-generator",
     )
+    p.add_argument(
+        '--use-https',
```

Comment:
I'd much rather if this was called something like "--pull-via-https", so that it's more obvious that it's related to pulling. Also please provide help and ideally an environment variable to be able to override this, too.